### PR TITLE
Add solace getting started guide and fix ambiguous content

### PR DESCRIPTION
**Pull request by Sarangan0219**
## Purpose

- Fixes https://github.com/wso2/docs-apim/issues/5641

### CODE REVIEW

**Review by ruthryi — APPROVED** [review]

File: en/docs/tutorials/integrating-with-solace.md [hunk 1]
```diff
@@ -35,8 +31,8 @@ WSO2 API Manager now supports integrating external third party Gateways and Brok
         show_as_token_endpoint_url = false
 
         [apim.gateway.environment.properties]
-        Organization = <Password of the Solace broker account>
-        DevAccountName = <Password of the Solace broker account>
+        Organization = <Organization created in SOlace>
```
Comment (wasuradananjith) [inline en/docs/tutorials/integrating-with-solace.md:34]:
```suggestion
        Organization = <Organization created in Solace>
```

File: en/docs/tutorials/integrating-with-solace.md [hunk 2]
```diff
@@ -115,7 +108,7 @@ The following are the two methods available in the Developer Portal to subscribe
 Note that the artifacts are deployed in the Solace broker, therefore API level and application-level throttling will not be supported for the Solace API subscriptions.
     
 !!! note
-    The Applications subscribed to Solace APIs have pre-generated production keys. Therefore, only the corresponding subscription will be made in solace broker. If you have not generated any keys such as Sandboxkeys, then subscriptions will not be made in solace broker.
+    The Applications subscribed to Solace APIs have pre-generated production keys. Therefore, only the corresponding subscription will be made in solace broker. If you have not generated any keys such as Sandboxkeys, then subscriptions will not be made in solace broker.  If you subscribed to an application without production keys and if you generate the production keys then a subscription is created internally.
```
Comment (wasuradananjith) [inline en/docs/tutorials/integrating-with-solace.md:111]:
```suggestion
    The Applications subscribed to Solace APIs have pre-generated Production keys. Therefore, only the corresponding subscription will be made in the Solace broker. If you have not generated any keys such as Sandbox keys, then subscriptions will not be made in the Solace broker.  If you subscribed to an application without Production keys and if you generate the Production keys then a subscription is created internally.
```
Shall we rephrase this like above?

Comment (Sarangan0219) [reply en/docs/tutorials/integrating-with-solace.md]: Fixed https://github.com/wso2/docs-apim/pull/5652/commits/bc8d2d35e96a6445d5d4c8f038040387c9db3e8c

File: en/docs/tutorials/integrating-with-solace.md (hunk 1 above)
Comment (Sarangan0219) [reply en/docs/tutorials/integrating-with-solace.md]: Fixed https://github.com/wso2/docs-apim/pull/5652/commits/72d996d13a19e8dbdb5c8ef4fae2b2282b757af9

**Review by wasuradananjith — APPROVED** [review]

**Pull request merged**